source/portable/NetworkInterface/DriverSAM/NetworkInterface.c

```diff
@@ -706,7 +725,7 @@ static BaseType_t prvGMACInit( NetworkInterface_t * pxInterface )
                     ucMACAddress[ 3 ] = pxEndPoint->ipv6_settings.xIPAddress.ucBytes[ 13 ];
```

Comment:
When adding IPv6 multi-cast MAC addresses of the IPv6 endpoints to the hash table, shouldn't it be of the form - [Address Mapping -- Multicast](https://datatracker.ietf.org/doc/html/rfc2464#section-7)

Reply:
This looks like a valid issue and the field ucMACAddress[ 2 ] needs to be set as well.

Reply:
Change will be reverted, see [Solicited-node multicast address](https://en.wikipedia.org/wiki/Solicited-node_multicast_address). It is used to give the DUT a multicast address which is used during the SLAAC for DAD (Duplicate Address Detection).